pycbc/conversions.py

```diff
@@ -796,9 +796,10 @@ def snr_from_loglr(loglr):
     if singleval:
         loglr = numpy.array([loglr])
     # temporarily quiet sqrt(-1) warnings
-    numpysettings = numpy.seterr(invalid='ignore')
+    orig_settings = numpy.geterr()
+    numpy.seterr(invalid='ignore')
     snrs = numpy.sqrt(2*loglr)
-    numpy.seterr(**numpysettings)
+    numpy.seterr(**orig_settings)
```

Comment:
Sorry to barge in, but I think you can simplify this with [`numpy.errstate`](https://numpy.org/doc/stable/reference/generated/numpy.errstate.html):

```python
    with numpy.errstate(invalid="ignore"):
        snrs = numpy.sqrt(2 * loglr)
```

Reply:
This is a more useful solution than what StackOverflow managed to give me! Thanks, I'll change this now.